gusto/recovery/recovery_options.py

```diff
@@ -0,0 +1,119 @@
+
```

Comment:
I think we could still do with a docstring at the top of the file describing what this module contains, e.g.
"""Contains an object to generate the set of spaces used for the recovery wrapper."""